src/DocsParser.ts

```diff
@@ -146,6 +146,16 @@ export class DocsParser {
           'HTMLElement documentation should not be considered a class',
         );
       }
+      let electronProcess = { main: true, renderer: true };
+      for (let tk of tokens) {
+        if (tk.type == 'inline' && tk.content.indexOf('Process') != -1) {
```

Comment:
This token should _start_ with `Process` right, let's limit the scope of the search.  Also let's use strict equality here

```suggestion
        if (tk.type === 'inline' && tk.content.indexOf('Process') === 0) {
```